source/frontend/src/features/mapSideBar/acquisition/tabs/checklist/detail/AcquisitionChecklistView.tsx

```diff
@@ -38,10 +39,12 @@ export const AcquisitionChecklistView: React.FC<IAcquisitionChecklistViewProps>
   const checklist = acquisitionFile?.acquisitionFileChecklist || [];
   const lastUpdated = lastModifiedBy(checklist);
 
+  const statusSolver = new StatusUpdateSolver(acquisitionFile);
+
   return (
     <StyledSummarySection>
       <StyledEditWrapper className="mr-3 my-1">
-        {keycloak.hasClaim(Claims.ACQUISITION_EDIT) && acquisitionFile !== undefined ? (
```

Comment:
are we ok removing that undefined acquisitionFile check?

Reply:
the status solver does take into account the acq file being null/undefined